Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -20,6 +20,17 @@ internal static class ShardMapUtils
         /// </summary>
         internal static readonly string ConnectRetryCount = "ConnectRetryCount";
 
+        /// <summary>
+        /// SqlConnectionStringBuilder property that allows specifying
+        /// active directoty authentication to connect to SQL instance.
+        /// </summary>
+        internal static readonly string Authentication = "Authentication";
+
+        /// <summary>
+        /// String representation of SqlAuthenticationMethod.ActiveDirectoryIntegrated
+        /// </summary>
```

Comment:
<remarks>
SqlAuthenticationMethod.ActiveDirectoryIntegrated.ToString() cannot be used because it may not be available in the .NET framework version that we are running in.
</remarks>